Automattic/wp-calypso
# Remove Happy Chat

Main part of efforts to sunset Happy Chat and remove all its references: pdm0RZ-BU-p2.

Previously:
 - https://github.com/Automattic/wp-calypso/pull/77164
 - https://github.com/Automattic/wp-calypso/pull/77327
 - https://github.com/Automattic/wp-calypso/pull/78173
 - https://github.com/Automattic/wp-calypso/pull/78179
 - https://github.com/Automattic/wp-calypso/pull/78184
 - https://github.com/Automattic/wp-calypso/pull/78244
 - https://github.com/Automattic/wp-calypso/pull/78262
 - https://github.com/Automattic/wp-calypso/pull/78317
 - https://github.com/Automattic/wp-calypso/pull/78397

...at some point, it's time to rip off the band-aid ;)

## Proposed Changes

* Remove all possible references to `/happy-?chat/i` ;)
* Related things that got removed as well:\
   * `inline-help` - most of it could be removed, it was related to the old (pre-Help Center) floating FAB. But some parts are used in Help Center and `/home` to show support docs search results.
   * `client/me/help/help-contact` - old contact form that was superceded completely by Help Center in https://github.com/Automattic/wp-calypso/pull/76894.

## Testing Instructions

Honestly, nothing should change for users. They should be already using Help Center in 99% of cases (and for 1% Happy Chat was not loading anyway, cause no one was available).

So, I think most important is to test different use cases and environments, especially ones I'm not familiar with (Mobile? Jetpack Cloud? Atomic/ETK, etc.).

The failure of the "Happychat Plugin" build on TeamCity is expected - the code has been removed and as seen in https://github.com/Automattic/wp-calypso/pull/77151#issuecomment-1557286022, the build config will be removed from TeamCity once this lands.

## Pre-merge Checklist
